src/doc/texturesys.rst

```diff
@@ -209,15 +209,16 @@ UDIM tiled texture atlases. The substitutions will occur if the texture
 filename initially passed to `texture()` does not exist as a concrete file
 and contains one or more of the following substrings:
 
-========== ========================
-`<UDIM>`   1001 + utile + vtile*10
-`<u>`      utile
-`<v>`      vtile
-`<U>`      utile + 1
-`<V>`      vtile + 1
-`_u##v##`  utile, vtile
-`%(UDIM)d` 1001 + utile + vtile*10
-========== ========================
+==========  ================================================
+`<UDIM>`    1001 + utile + vtile*10
+`<u>`       utile
+`<v>`       vtile
+`<U>`       utile + 1
+`<V>`       vtile + 1
+`<UVTILE>`  equivalent to `u<U>_v<V>`
```

Comment:
This is a little confusing since `<U>` already implies the 'u' prefix -- I had to read the source code to know that though. Same for the other singular tokens like `<u>` etc.  It's really equivalent to `<U>_<V>`.

Reply:
Oh, you are right! I had forgotten that. I fill fix.